src/editor/koLanguageCommandHandler.p.py

```diff
@@ -2187,3 +2191,13 @@ def getLine(scin, lineNo):
     lineEnd = scin.getLineEndPosition(lineNo)
     line = scin.getTextRange(lineStart, lineEnd)
     return line
+
+def getReflowLine(scin, lineNo, view):
+    import reflow
+    line = reflow.Line(getLine(scin, lineNo));
```

Comment:
delete.

Reply:
...the comma obvs...not the whole line :D

Reply:
Of course I'd have at least one of them.

Reply:
I'd question your identity if there wasn't one.